access/ms-teams/configure.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"archive/zip"
+	"embed"
+	"fmt"
+	"html/template"
+	"io"
+	"os"
+	"path"
+
+	"github.com/google/uuid"
+	"github.com/gravitational/teleport-plugins/lib"
+	"github.com/gravitational/trace"
+)
+
+const (
+	guideURL = "https://goteleport.com/docs/enterprise/workflow/"
+)
+
+var (
+	//go:embed _tpl/teleport-ms-teams.toml
+	confTpl string
+
+	//go:embed _tpl/manifest.json
+	manifestTpl string
+
+	//go:embed _tpl/outline.png _tpl/color.png _tpl/teleport-ms-teams-role.yaml
+	assets embed.FS
+
+	// zipFiles represents file names which should be compressed into app.zip
+	zipFiles = []string{"manifest.json", "outline.png", "color.png"}
+)
+
+// payload represents template payload
+type payload struct {
+	AppID      string
+	AppSecret  string
+	TenantID   string
+	TeamsAppID string
+}
+
+// configure creates required template files
+func configure(targetDir, appID, appSecret, tenantID string) error {
+	var step byte = 1
+
+	p := payload{
+		AppID:      appID,
+		AppSecret:  appSecret,
+		TenantID:   tenantID,
+		TeamsAppID: uuid.New().String(),
+	}
+
+	lib.PrintVersion(appName, Version, Gitref)
+	fmt.Println()
+
+	_, err := os.Stat(targetDir)
+	if !os.IsNotExist(err) {
+		return trace.Wrap(err)
+	}
+
+	err = os.MkdirAll(targetDir, 0777)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	printStep(&step, "Created target directory: %s", targetDir)
+
+	renderTemplateTo(confTpl, p, path.Join(targetDir, "teleport-ms-teams.toml"))
+	renderTemplateTo(manifestTpl, p, path.Join(targetDir, "manifest.json"))
+
+	printStep(&step, "Generated configuration files")
+
+	a, err := assets.ReadDir("_tpl")
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	for _, a := range a {
```

Comment:
We have two variables named `a`
Can we please rename one of them, if possible with something more explicit